docs/hopsworksai/aws/eks_ecr_integration.md

```diff
@@ -192,10 +175,10 @@ selfLink: /api/v1/namespaces/kube-system/configmaps/aws-auth
 uid: c794b2d8-9f10-443d-9072-c65d0f2eb552
 ```
 
-Follow the example below (lines 13-16) to add your role to *mapRoles* and assign *system:masters* group to your role. Make sure to replace 'YOUR ROLE RoleARN' with your role RoleARN before saving.
+Follow the example below (lines 13-16) to add your role to *mapRoles* and assign the *system:masters* group to your role. Make sure to replace 'YOUR ROLE RoleARN' with the *Role ARN* you copied in the previous step before saving.
 
 !!! warning
-    You need to use the RoleARN not the instance profile ARN, also make sure to keep the same formatting as in the example below.
+    Make sure to keep the same formatting as in the example below. The configuration format is sensitive to indentation and copy-pasting does not always keep the correct indentation.
```

Comment:
We should also keep this "Make sure to use the RoleARN not the instance profile ARN"

Reply:
I'll check how to add it where I say to copy the RoleARN, but when I read this part I got more confused than helped. I was thinking that the "instance profile arn" was the role arn of the instance profile so I didn't understand which role ARN I had to copy. I only understood after searching for instance prifle arn. Maybe putting a picture of what needs to be copied will help more than saying what not to copy.

Reply:
I agree a screenshot then would make more sense

Reply:
When you check the Instance profile role in AWS, you will find two RoleARNs
`Role ARN` and `Instance Profile ARNs`, we should use the `Role ARN`

Reply:
<img width="1147" alt="Screenshot 2021-03-23 at 13 53 49" src="https://user-images.githubusercontent.com/1159906/112150224-2420da00-8be0-11eb-8fbd-805724d76b9f.png">